org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/telemetry/LinkTelemetry.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SonarLint for Eclipse
+ * Copyright (C) 2015-2023 SonarSource SA
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.eclipse.core.internal.telemetry;
+
+public enum LinkTelemetry {
+
+  CONNECTED_MODE_DOCS("connectedModeDocs", "https://docs.sonarsource.com/sonarlint/eclipse/team-features/connected-mode"),
```

Comment:
The problem I have is that the UI package is out of reach from core. So I moved the SonarLintDoc to the core package. It's cleaner because every links are now in this class, but the package is in the core 🤷🏻 

Let me know what you think about it

Reply:
I think that's good 👍